server2/Arcadia.Assistant/Arcadia.Assistant.ExternalStorages.SharepointOnline/SharepointApiModels/SharepointListItemRequest.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Arcadia.Assistant.ExternalStorages.SharepointOnline.SharepointApiModels
+{
+    using System.Runtime.Serialization;
+
+    using Abstractions;
+
+    [DataContract]
+    public class SharepointListItemRequest : StorageItem
```

Comment:
It is not the same as `StorageItem`. Previous default values of this class properties and default values of `StorageItem` properties differs, here `DataContract` and `DataMember` attributes were used against `JsonPropertyName` in `StorageItem`.

Reply:
We can remove redundant [DataContract] and [DataMember] attributes.

Reply:
Removed